components/authz-service/storage/v2/postgres/postgres.go

```diff
@@ -1161,6 +1161,50 @@ func (p *pg) ListRules(ctx context.Context) ([]*v2.Rule, error) {
 	return rules, nil
 }
 
+func (p *pg) ListRulesForProject(ctx context.Context, projectID string) ([]*v2.Rule, error) {
+	projectsFilter, err := projectsListFromContext(ctx)
+	if err != nil {
+		return nil, p.processError(err)
+	}
+
+	ctx, cancel := context.WithCancel(ctx)
```

Comment:
Wouldn't we want to move this to the top? (above https://github.com/chef/automate/pull/498/files#diff-d28b0cc9c342bf1da873bddba25960f6R1165)